src/components/ContactsList/Contacts/ContactName.jsx

```diff
@@ -1,11 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
+import { withStyles } from '@material-ui/core/styles'
 
-const ContactName = ({ displayName, familyName }) => {
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+const customStyles = () => ({
+  root: {
+    fontWeight: 'normal'
+  }
+})
+
+const ContactName = ({ classes, displayName, familyName }) => {
   const namesToDisplay = (displayName && displayName.split(' ')) || []
   return (
-    <div className="u-ellipsis u-ml-1">
+    <Typography
+      variant="h6"
```

Comment:
Is it not possible to use body1 or body2 ? Why do we have to add a font-weight: normal ?

Reply:
🤦 body1 works fine